index.html

```diff
@@ -173,6 +173,30 @@ <h2>Calls for implementation and feedback</h2>
 
 <p>When an addition is accepted at the “candidate” (stage 3) maturity level, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage and external feedback.
 
+<h2>Withholding Consensus for Advancement</h2>
+
+<p>During the discussion of a proposal, at any stage, any aspect may be discussed, including those outside of the scope of its current stage. A proposal’s advancement may not have consensus, and may be "blocked". A block is a forumated reason for why a proposal should not advance in it's present form. The committee does not have a concept of a rejected proposal.</p>
+
+<p>In withholding consensus for advancement, a reason should be stated either as a delay or as a block. If no reason is given, the objection is void for vagueness. In the case of a delay -- a delegate may need more time to consider a given aspect of a proposal. They may request additional time, or allow conditional advancement.</p>
+
+<p>Conditional advancement is time-limited, giving the person raising the block time to discuss with the champions and authors about their concerns. If a proposal has conditional advancement, an issue is opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.</p>
```

Comment:
```suggestion
<p>Conditional advancement is time-limited, giving the person raising the block time to discuss with the champions and authors about their concerns. If a proposal has a conditional advancement, an issue must be opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.</p>
```

I miss a last part of this paragraph, like: "the proposal does not advance and the committee should discuss alternatives". Anything actionable would be great.

Reply:
I am not sure that the committee would be responsible for finding alternatives, maybe the champions?